proxy/hdrs/HTTP.cc

```diff
@@ -625,7 +625,27 @@ http_hdr_type_set(HTTPHdrImpl *hh, HTTPType type)
 /*-------------------------------------------------------------------------
   -------------------------------------------------------------------------*/
 
-void
+bool
+is_version_supported(const uint8_t major, const uint8_t minor)
+{
+  if (major == 1) {
+    return minor == 0 || minor == 1;
+  }
+
+  if (major == 2) {
+    return minor == 0;
+  }
+
+  return false;
+}
+
+bool
+is_http_hdr_version_supported(const HTTPVersion &http_version)
+{
+  return is_version_supported(http_version.get_major(), http_version.get_minor());
+}
+
+bool
```

Comment:
This function doesn't return a value.

Reply:
Ugh, copy paste error from internal repo to ASF. Fixed.